cpp/models/abm/world.cpp

```diff
@@ -100,6 +100,9 @@ void World::migration(TimePoint t, TimeSpan dt)
             std::make_pair(&go_to_icu, std::vector<LocationType>{LocationType::Hospital, LocationType::ICU}),
             std::make_pair(&go_to_quarantine, std::vector<LocationType>{LocationType::Home})};
     }
+
+    update_testing_scheme_activity_status(t);
```

Comment:
Why is this here? Isn't this something that needs to be in world::evolve? this has nothing to do with migration